healdata.org/manifests/hatchery/hatchery.json

```diff
@@ -391,6 +391,12 @@
       "aws_account_id": "252835817747",
       "name": "healworkspaces",
       "region": "us-east-1"
+    },
+    {
+      "user_id": "ahyde@arthritis.org",
```

Comment:
What login method will she be using?

Reply:
good Q, but i don't know if we know that? @hughestr